docs/marketplace/add_to_marketplace/add_item_by_type/add_custom_resource.md

```diff
@@ -0,0 +1,256 @@
+---
+id: add_custom_resource
+title: Publish your custom resources to the Marketplace
+sidebar_label: Create a custom resource
+---
+
+It is possible to include in the Marketplace Kubernetes [custom resources](/console/design-your-projects/custom-resources.md) to extend the Kubernetes API or include your own API into the project.
+
+They are basically defined by two fields:
+
+- the `apiVersion` field, which is the API version of the resource
+- the `kind` field, which is the kind of the resource
+
+Custom resources can be added to your project by creating one anew or by installing an existing one from the Marketplace, using the `miactl` tool or via _CMS_. They are defined with the usual marketplace item schema, by having the _type_ field equal to `custom-resource`.
```

Comment:
"They are defined with the usual marketplace item schema"

What is the usual schema? If i'm reading this document for the first time I may not familiar with this, we should link some examples